packages/atomic/src/locales.json

```diff
@@ -1125,5 +1125,17 @@
   "stars-range": {
     "en": "{{value}} to {{count}} stars",
     "fr": "{{value}} à {{count}} étoiles"
+  },
+  "with-colon": {
+    "en": "{{text}}:",
+    "fr": "{{text}}\u00A0:"
```

Comment:
`\u00A0` is a non-breaking space (`&nbsp;`).

I added them because I recall seeing spaces being added before colons, quotation marks and parentheses often in French, but I'm uncertain whether they belong here.

Reply:
@btaillon that's a quite intriguing thing to discuss :) 

As I know, there's no such a rule in Canadian French. 
On the contrary, there IS a rule about non-breaking spaces in European French. The rule says there should be a non-breaking-space BEFORE and AFTER the following punctuation marks:
- exclamation marks
- question marks
- semicolons
- colons
- the percentage mark
- currency symbols
- the hash
- the guillemet

So, I'd like to summon @jpmarceau here so he can ask my question: do we use Canadian French or European French in our public texts (UI, docs, etc.)? (_I know for sure that we use the American version of English, but not aware about French, unfortunately._)

If we are using Canadian French, we don't need non-breaking spaces at all.
If we are using European French, we do need non-breaking spaces.

Reply:
I'm not sure about which French we use exactly, but I did a bit of research and it seems that even here in Quebec we put an insecable whitespace before colons http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2039
So I think this is fine 👍

Reply:
Wow! I'm saving that link, very useful table.

Reply:
@jpmarceau thx for research! Doesn't it mean that we need adding the second non-breaking space after the colon?